doc/nrf/protocols/matter/getting_started/low_power_configuration.rst

```diff
@@ -19,6 +19,65 @@ The following Matter samples and applications use the low power configuration by
 * :ref:`Matter window covering sample <matter_window_covering_sample>`
 * :ref:`Matter weather station application <matter_weather_station_app>`
 
+.. _ug_matter_device_low_power_icd:
+
+Enable Matter Intermittently Connected Devices support
+******************************************************
+
+The Matter treats power-optimized devices as a part of the wider group, which are devices that are intermittently connected to the network or unreachable for periods of time.
+So-called Intermittently Connected Devices (ICD) can be unreachable for various reasons, however such details are out of Matter protocol consideration.
+The ICD management layer is agnostic to underlying network layer and focuses on managing the device reachability in a high level way.
+
+To enable the ICD support in Matter, set the :kconfig:option:`CONFIG_CHIP_ENABLE_ICD_SUPPORT` Kconfig option to ``y``.
+
+.. _ug_matter_device_low_power_icd_modes:
+
+Operating modes
+===============
+
+An ICD can operate in one of two states that are idle and active mode.
+Idle mode is a state, where the device is unreachable for certain period of time that maximum value is specified by the idle mode duration (previously called idle mode interval) parameter.
+In the idle mode, the device configures the networking interface in a slow polling, which means it is able to receive messages with the fastest frequency of slow polling interval.
+Active mode is a state, where the device is reachable and will answer in a timely manner for minimum period of time specified by the active mode duration (previously called active mode interval) parameter.
+In the active mode, the device configures the networking interface in a fast polling, which means it is able to receive messages with the fastest frequency of fast polling interval.
+The client of ICD has to respect the slow polling and fast polling intervals and should not try to send messages with the bigger frequency.
+
+To configure the ICD communication intervals, use the following Kconfig options:
+
+* :kconfig:option:`CONFIG_CHIP_ICD_IDLE_MODE_DURATION` to set idle mode duration value in seconds (for example, ``120``).
+* :kconfig:option:`CONFIG_CHIP_ICD_ACTIVE_MODE_DURATION` to set active mode duration value in milliseconds (for example, ``300``).
+* :kconfig:option:`CONFIG_CHIP_ICD_SLOW_POLL_INTERVAL` to set slow polling interval value in milliseconds (for example, ``5000``).
+* :kconfig:option:`CONFIG_CHIP_ICD_FAST_POLLING_INTERVAL` to set fast polling interval value in milliseconds (for example, ``200``).
+
+An ICD switches modes from idle to active for various reasons that are:
```

Comment:
Good point. If the list is not exhaustive, the line can end with ", such as:"

Reply:
Actually I think these are the only reasons copied from Matter spec, because it treats "an application event" as everything else specific to the application. Nevertheless I think adding "such as" statement will not harm.